tests/hikari/impl/test_entity_factory.py

```diff
@@ -6498,6 +6498,7 @@ def test_deserialize_template(
         assert template.creator == entity_factory_impl.deserialize_user(user_payload)
         assert template.created_at == datetime.datetime(2020, 12, 15, 1, 54, 35, tzinfo=datetime.timezone.utc)
         assert template.updated_at == datetime.datetime(2020, 12, 15, 1, 57, 35, tzinfo=datetime.timezone.utc)
+        assert template.app is mock_app
```

Comment:
Nitpick, but for consistently, this should be the first attribute assert

Reply:
Done <3